web-components/src/components/table/index.tsx

```diff
@@ -1,298 +1,94 @@
-          {stableSort(rows, getComparator(order, orderBy))
-            // .slice(page * rowsPerPage, page * rowsPerPage + rowsPerPage)
-            .map((row, index) => {
-              const labelId = `enhanced-table-checkbox-${index}`;
-              const hasViewOnLedger =
-                row.eventByEventId?.creditVintageByEventId?.txHash && txClient && onViewOnLedger;
-              return (
-                <TableRow
-                  tabIndex={-1}
-                  key={row.name}
-                  className={clsx(classes.row, canClickRow && classes.rowClickable)}
-                  onClick={() => handleClickNavigate(row.url)}
-                >
-                  <TableCell className={clsx(classes.cell, classes.nameCell)} id={labelId} scope="row">
-                    <div className={classes.name}>
-                      <DocumentIcon className={classes.icon} fileType={row?.name?.split('.')?.pop()} />{' '}
-                      {row.name}
-                    </div>
-                  </TableCell>
-                  <TableCell className={classes.cell} align="left">
-                    {row.type}
-                  </TableCell>
-                  <TableCell className={classes.cell} align="left">
-                    {typeof row.date === 'string' && getFormattedDate(row.date, options)}
-                  </TableCell>
-                  {hasViewOnLedger && onViewOnLedger && (
-                    <TableCell className={clsx(classes.cell, classes.documentCell)} align="right">
-                      <ContainedButton
-                        className={clsx(classes.button, classes.ledgerBtn)}
-                        onClick={() => onViewOnLedger(row.eventByEventId.creditVintageByEventId)}
-                        startIcon={<ShieldIcon />}
-                      >
-                        view on ledger
-                      </ContainedButton>
-                    </TableCell>
-                  )}
-                  {hasViewOnLedgerColumn && !hasViewOnLedger && (
-                    <TableCell className={clsx(classes.cell, classes.documentCell)} align="right"></TableCell>
-                  )}
-                  <TableCell className={clsx(classes.cell, classes.documentCell)} align="right">
-                    <a href={row.url} target="_blank" rel="noopener noreferrer" className={classes.link}>
-                      <OutlinedButton startIcon={<EyeIcon />} className={classes.button}>
-                        view document
-                      </OutlinedButton>
-                    </a>
-                  </TableCell>
-                </TableRow>
-              );
-            })}
-          {/*emptyRows > 0 && (
-            <TableRow style={{ height: (dense ? 33 : 53) * emptyRows }}>
-              <TableCell colSpan={6} />
-            </TableRow>
-          )*/}
-        </TableBody>
-      </Table>
-    </TableContainer>
-  );
-}
+export { StyledTableContainer, StyledTableRow, StyledTableCell, StyledTableSortLabel };
```

Comment:
I could separate all these components into different files, but since the styles are so intertwined, it made sense to just export them together from the main file. I will go with whatever y'all prefer.